docs/sources/getting-started/_index.md

```diff
@@ -1,14 +1,141 @@
+    ```bash
+    mkdir evaluate-loki
+    cd evaluate-loki
+    ```
+1. Download `loki-config.yaml`, `promtail-local-config.yaml`, and `docker-compose.yaml`:
+
+    ```bash
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/loki-config.yaml -O loki-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/promtail-local-config.yaml -O promtail-local-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/docker-compose.yaml -O docker-compose.yaml
+    ```
+
+## Deploy the test environment
+
+All shell commands are issued from the `evaluate-loki` directory.
+
+1. With `evaluate-loki` as the current working directory, deploy the test environment using `docker-compose`:
+    ```bash
+    docker-compose up
+    ```
+1. (Optional) Verify that the Loki cluster is up and running. The read component returns `ready` when you point a web browser at http://localhost:3101/ready. The message `Query Frontend not ready: not ready: number of schedulers this worker is connected to is 0` will show prior to the read component being ready.
+The write component returns `ready` when you point a web browser at http://localhost:3102/ready. The message `Ingester not ready: waiting for 15s after being ready` will show prior to the write component being ready.
+
+## Use Grafana and the test environment
+
+Use [Grafana](https://grafana.com/docs/grafana/latest/) to query and observe the log lines captured in the Loki cluster by navigating a browser to http://localhost:3000.
+The Grafana instance has Loki configured as a [datasource](https://grafana.com/docs/grafana/latest/datasources/loki/).
+
+Click on the Grafana instance's [Explore](https://grafana.com/docs/grafana/latest/explore/) icon to bring up the explore pane.
+
+Use the Explore dropdown menu to choose the Loki datasource and bring up the Loki query browser.
+
+Try some queries.
+Enter your query into the **Log browser** box, and click on the blue **Run query** button.
+
+To see all the log lines that flog has generated:
+```
+{container="evaluate-loki_flog_1"}
+```
+
+The flog app will generate log lines for invented HTTP requests.
+To see all `GET` log lines, enter the query:
+
+```
+{container="evaluate-loki_flog_1"} |= "GET"
+```
+For `POST` methods:
+```
+{container="evaluate-loki_flog_1"} |= "POST"
+```
+
+To see every log line with the value of 401: 
+```
+{container="evaluate-loki_flog_1"} |= "401"
+
+```
+To see every log line other than those that contain the value 401: 
+```
+{container="evaluate-loki_flog_1"} != "401"
+```
+
+## Stop and clean up the test environment
+
+To break down the test environment:
+
+- Close the Grafana browser window
+
+- Stop and remove all the Docker containers. With `evaluate-loki` as the current working directory:
+    ```bash
+    docker-compose down
+    ```
+
+## Modifying the flog app output
```

Comment:
Hmmm I think this subsection seems out-of-place/looks unnecessary. WDYT?

Reply:
Let's see what others say in their reviews.  I wanted this effort to be a guide, not a tutorial.  Part of making a guide is providing paths for extending what is given to what is useful for the readers.

I envision adding another section (in the future, when time allows) which shows how to run Promtail and the app outside of Docker containers.  This would promote understanding of how to set up Promtail configuration.

So, in the future, I think there will be other sections that we might classify as extending the test environment to further evaluation of Loki.  I think that we might even use this setup (maybe replace flog with another app?) to be used as a LogQL query-writing practice area.